### PR TITLE
[Announcement] Add welcome modal for users coming from explore.va.gov

### PR DESCRIPTION
**Pull request by ncksllvn**
## Description
This PR adds an announcement for users who are redirect from `explore.va.gov`. Currently, there are no redirects in place, but there will be soon. This can be tested by adding the query param `from=explore.va.gov` into the URL of any page.

Ticket - https://github.com/department-of-veterans-affairs/vets.gov-team/issues/18788

## Testing done
All local

## Screenshots
`/careers-employment/?from=explore.va.gov`

image

## Acceptance criteria
- [x] Explore-VA Welcome Modal opens
- [x] Optimized image via Tiny PNG

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/platform/site-wide/announcements/components/ExploreVAModal.jsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+          This is part of our continued work to make it easier for you to access
+          and manage the benefits you've earned.
+        </p>
+        <button type="button" onClick={dismiss}>
+          Continue to VA.gov
+        </button>
+      </Modal>
+    );
+  }
+}
```
Comment (ncksllvn) [inline src/platform/site-wide/announcements/components/ExploreVAModal.jsx]: This largely just copied the Vets.gov Welcome Modal.

**Comment by ncksllvn** [comment]
Hey all, you can see this working by visiting https://vetsgov-pr-10254.herokuapp.com/?from=explore.va.gov.

**Comment by jbalboni** [comment]
@ncksllvn When I clicked on the Heroku link above, I got the explore modal and then the default welcome modal after I clicked through. Is that intentional?

**Comment by ncksllvn** [comment]
@jbalboni Ugh, yeah, that's expected from a dev perspective because that is how announcements work, but this probably warrants changing that to only render one announcement per each page load

**Comment by ncksllvn** [comment]
@jbalboni I added the regular VA Welcome Modal as a related announcement, so that it's also dismissed after the user sees the Explore VA Modal since it's kinda weird to greet the user twice.

Thanks for catching that!

**Review by jbalboni — APPROVED** [review]

**Pull request merged**